extensions/amp-social-share/1.0/README.md

```diff
@@ -0,0 +1,597 @@
+---
+$category@: ads-analytics
+formats:
+  - websites
```

Comment:
Do we need teaser text here?

Reply:
Oh yeah, good call.

Reply:
oh, i'd meant it was missing those attributes. should we be removing the frontmatter from the `<extension>/1.0/README.md` files?

Reply:
Yeah, I learned this morning that they aren't needed so I created #36121 to fix everything so far